research-software-directory/RSD-as-a-service
# Switch from Yarn to npm

## Switch from Yarn to npm

Changes proposed in this pull request:

* Change all `yarn` commands to their equivalent `npm` commands
* Remove all references to Yarn in the documentation

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Check if everything still works
* Do a global search in the project to see if all references to Yarn are removed (it is still used in some of `**/packaage-lock.json`)

Closes #1278
Related to #843

PR Checklist:
